jobserver/models.py

```diff
@@ -5,6 +5,7 @@
 from django.contrib.auth.models import AbstractUser
 from django.db import models
 from django.urls import reverse
+from first import first
```

Comment:
I'm open to being convinced but I'm not especially in favour of requiring a whole new package whose implementation is at most 3 lines of python.  I mean, I agree it's a missing function, it's just I also have a preference for trivial code being completely under our control

Reply:
[10](https://github.com/hynek/first/blob/master/first.py#L69-L78) if you ignore the whitespace and include the declaration!

There was talk about adding it to the stdlib, but it's not implmenented in C so didn't fit in itertools.  As a project it's very stable, the code hasn't had any updates since 2012.

I've always treated it as a default package, I end up reaching for it in everything I work on because it's handles a bit of boiler plate (try/catch on IndexError), and its `key` kwarg makes it useful to filter and index in a single function call.

Reply:
`¯\_(ツ)_/¯`